elements/src/main/java/com/manydesigns/elements/crypto/KeyManager.java

```diff
@@ -170,8 +177,16 @@ private String getPassPhrase(String passphrasePath) throws IOException, InvalidP
                 passPhrase.append(line);
             }
         } catch (IOException e) {
-            logger.error("getPassPhrase: " + e.getMessage(), e);
-            throw e;
+            logger.warn("getPassPhrase: " + e.getMessage());
```

Comment:
Non conosco la logica di questa parte, ma mi sembra sospetto usare la variabile di ambiente solo in caso di eccezione, da utente mi aspetterei che se la variabile d'ambiente c'è venga utilizzata.

Reply:
L'ho fatto perchè, prima guardo se esiste il file uso il file, altrimenti cerco la variabile d'ambiente....
Secondo te ha senso che vinca la variabile d'ambiente rispetto al file?

Reply:
Io da utente mi aspetterei che se setto la variabile d'ambiente prevalga quella, magari fai un sondaggio e vedi che ne pensano gli altri :)